gulp-tasks/move.js

```diff
@@ -21,15 +21,15 @@ module.exports = {
   // Moves JavaScript.
   js: function() {
     return gulp.src([
-      './src/{global,layout,components}/**/*.js',
-      '!./src/{global,layout,components}/**/*.es6.js'
+      './src/{global,components}/**/*.js',
+      '!./src/{global,components}/**/*.es6.js'
     ], { base: './' })
       .pipe(rename(function (path) {
         path.dirname = '';
         return path;
       }))
       .pipe(gulp.dest('./dist/js'));
-  },
+  },   
```

Comment:
@mark-casias some compilers add a new line to the end of files. maybe that was the case here